### DISCUSSION TITLE
So, I did a thing... And Thank you!

### DISCUSSION THREAD

**Discussion by sysRoot** [timlrx/tailwind-nextjs-starter-blog#253] [Show and tell]
I really much liked the design and the functionality, and while I very much appreciate all the markdown and mdx support, I actually retrofitted my copy to work with Strapi, and used MeiliSearch/Algolia Instant search to provide a more robust search experience for the Strapi backend running with GraphQL against MariaDB. I selected your project as there isn't much in the way of general user pre-made Tailwind designs.

I did however check out and toy with the project before stripping away what wasn't useful for my goal.
A+ Work, and I will include a blog post on my site about the original design and how it works rudimentary-wise. 
I don't expect technical readers, but I also left the Link to this Github repo intact in case anyone does decide to check it out.

A list of changes I made to work within Strapi's confines of (default reliable max) returning a 100 results per query, considering I modified this not just for blogs but for poetry.

- Split a new type up and mirrored the design of "blog articles" to "poems"
- Conjoined the tags onto one page, however each has their own dynamic route and are also paginated as well
- Made adequate replication and modifications to ensure both types route and display similarly.
- Because of the way I set it up, I set each result to only be 5 deep per query, hence paginating tags
- Implemented a modal search, which I'm still tweaking because my CSS skill work is awful.
- Added a more drop down, which links to a page for my book, and donation page, and a page linking my terrible soundcloud songs
- Refashioned "Projects" into a "Friendlies" page.

I will be making a donation towards your work, as it is excellent, and really well done, and I felt very awful taking apart a lot of the hard work just to make use of the tailwind templating and its use of NextJS.

I guess what I'm trying to say is thanks, thank you very much for your hard work, and making this opensource. 🙏🙏🙏

If interested, my awful hack job is visible at [meaningpoetry.com](https://meaningpoetry.com)

**Comment by timlrx** [comment]
Thanks for the kind words! Great that you managed to retrofit it for your own backend of choice. I decided to go with markdown instead of other CMS as it is the most straightforward way to deploy and manage, but nice knowing that you managed to get it working with Strapi as well.

The project takes lots of inspiration, design and code wise from the main tailwindcss blog and lee robinson's blog among others - glad that you find it useful :)
